src/block-cursor.ts

```diff
@@ -170,6 +170,11 @@ function measureCursor(cm: CodeMirror, view: EditorView, cursor: SelectionRange,
     }
     let style = getComputedStyle(node as HTMLElement);
     let left = pos.left;
+    // TODO remove coordsAtPos when all supported versions of codemirror have coordsForChar api
```

Comment:
how far back are we planning on supporting?

Reply:
I am not sure. Maybe until we accumulate enough changes to bump the major version?